plugins/plugin-languageserver/che-plugin-languageserver-ide/pom.xml

```diff
@@ -149,7 +149,7 @@
                 <version>${project.version}</version>
                 <executions>
                     <execution>
-                        <phase>process-sources</phase>
+                        <phase>generate-sources</phase>
```

Comment:
>It's impossible to build in offline mode when the local maven cache is empty

Ok. I wasn't really clear.  I mean remove only ```org.eclipse.che``` artifacts or to be more precise ```che-plugin-languageserver-ide``` and when rebuilt  ```che-plugin-languageserver-ide``` in offline mode. It's should be possible. Same situation we have when maven release plugin is releasing.